### If you want the fancy proctitles, then you need to enable
### CONFIG_CHECKPOINT_RESTORE in your kernel.
--- a/src/lxc/utils.c
+++ b/src/lxc/utils.c
@@ -1458,7 +1458,7 @@ int setproctitle(char *title)
 	if (ret == 0)
 		strcpy((char*)arg_start, title);
 	else
-		SYSERROR("setting cmdline failed");
+		INFO("setting cmdline failed - %s", strerror(errno));
 
 	return ret;
 }
